src/message.ts

```diff
@@ -613,6 +625,45 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo UserId/RoomId/Room/Contact
+   */
+  public forward(room: Room): Promise<boolean>
+  public forward(contact: Contact): Promise<boolean>
+  public forward(id: string): Promise<boolean>
```

Comment:
Send to `UserId` should not be supported.

The API of `Message` class should be abstracted with the `Room` and `Contact`, but not the `UserName` or other `id`.